doc/user_guide/eds.rst

```diff
@@ -577,22 +577,22 @@ background.
     Zr_Lb3
     background_order_6
 
-The width and the energies are fixed, while the heights of the sub-X-ray
-lines are linked to the main X-ray lines (alpha lines). The model can now be
-fitted:
-
-.. code-block:: python
-
-    >>> m.fit()
-
 The background fitting can be improved with
```

Comment:
With these changes, this wording needs to be changed too, as it is not "improved" from a previous fit anymore.

Reply:
I have committed the changes to my forked EDX_doc branch, I have renamed this branch to "hyperspy next_rebased" to reduce confusion, and rebased it on the "next_patch" branch. I tried to solved the conflicts and hope it can work!